packages/block-library/src/separator/edit.js

```diff
@@ -29,19 +28,10 @@ function SeparatorEdit( { color, setColor, className } ) {
 					color: color.color,
 				} }
 			/>
-			<InspectorControls>
-				<PanelColorSettings
-					title={ __( 'Color Settings' ) }
-					colorSettings={ [
-						{
-							value: color.color,
-							onChange: setColor,
-							label: __( 'Color' ),
-						},
-					] }
-				>
-				</PanelColorSettings>
-			</InspectorControls>
+			<SeparatorSettings
```

Comment:
The following might be easier to follow for those not familiar with the internal implementation:
```suggestion
			<SeparatorInspectorConrols
```